Utils/main.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+
+from helper_functions import find_median, open_file, plot_histogram
+
+
+# define the main function for the program to run from here
+def main(
+    file_path: str,
+    feature_name: str,
+    image_name="image_name",
+    median=True,
+    histogram=True,
+    bins=50,
+):
+
```

Comment:
Consider providing a docstring to help describe each of the parameters and the purpose of this function.